doc/release/release_0.16.rst

```diff
@@ -0,0 +1,384 @@
+Announcement: scikit-image 0.16.0
+=================================
+
+We're happy to announce the release of scikit-image v0.16.0!
+
+scikit-image is an image processing toolbox for SciPy that includes algorithms
+for segmentation, geometric transformations, color space manipulation,
+analysis, filtering, morphology, feature detection, and more.
+
+For more information, examples, and documentation, please visit our website:
+
+https://scikit-image.org
+
+Starting from this release, scikit-image will follow the spirit of the recently
+introduced numpy deprecation policy -- NEP 29
+(https://github.com/numpy/numpy/blob/master/doc/neps/nep-0029-deprecation_policy.rst). 
+Accordingly, scikit-image 0.16 drops support for Python 3.5.
+This release of scikit-image officially supports Python 3.6 and 3.7.
+
+New Features
+------------
+- Introduced a new `skimage.evaluate` module containing simple metrics (mse,
+  nrme, psd) and segmentation metrics (adapted rand error, variation of
+  information) (#4025)
+- Added TV-L1 optical flow algorithm for registration --
+  `skimage.registration.optical_flow_tvl1` (#3983)
+- Added a function drawing a line in n-dimensional space -- `skimage.draw.line_nd`
+  (#2043)
+- Added Farid & Simoncelli edge filters - `skimage.filters.farid`,
+  `skimage.filters.farid_h`, and `skimage.filters.farid_v` (#3775)
+- Added Majority voting filter assigning to each pixel the most commonly occuring value
+  within its neighborhood -- `skimage.filters.majority` (#3836, #3839)
+- Added multi-level threshold "multi-Otsu" method, a thresholding algorithm
+  used to separate the pixels of an input image into several classes by
+  maximizing the variances between classes --
+  `skimage.filters.threshold_multiotsu` (#3872, #4174)
+- Added new example data -- `skimage.data.shepp_logan_phantom`, `skimage.data.colorwheel`,
+  `skimage.data.brick`, `skimage.data.grass`, `skimage.data.roughwall`, `skimage.data.cell`
+  (#3958, #3966)
+- Added function to find labeled image properties and convert them to
+  dictionnary -- `skimage.measure.regionprops_table` (#3959)
```

Comment:
typo:  
dictionnary -> a dictionary

Reply:
Also there are a mixture of single and double-backticks around module / function names throughout this file (e.g. `skimage.measure.regionprops_table` here). Should they all be double backtick?

Reply:
single backticks allow to automatically link to the module/function documentation. Double back-ticks are for everything else. I tried to use single backticks where it made sense and double backticks when it was something that isn't documented (argument name for example)

Reply:
Okay, thanks for the explanation. 

Do you happen to know what sphinx configuration setting gets the single-backticks to become links as opposed to just italics? I was trying to get links from single backticks recently in `PyWavelets`, but it was always just rendering in italics. I had to use :func:`function_name` instead of just single backticks alone for it to work. I assume this may be some configuration option difference, but hadn't had time to look into it yet.

Reply:
It's `default_role = "autolink"` in the `conf.py`

Reply:
(As a side note, if it doesn't find the function/module, it does put it in italic)